src/host/pico_multicore/include/pico/multicore.h

```diff
@@ -13,29 +13,28 @@
 extern "C" {
 #endif
 
-void multicore_reset_core1();
+void multicore_reset_core1(void);
 void multicore_launch_core1(void (*entry)(void));
 void multicore_launch_core1_with_stack(void (*entry)(void), uint32_t *stack_bottom, size_t stack_size_bytes);
-void multicore_sleep_core1();
 void multicore_launch_core1_raw(void (*entry)(void), uint32_t *sp, uint32_t vector_table);
 
-bool multicore_fifo_rvalid();
-bool multicore_fifo_wready();
+bool multicore_fifo_rvalid(void);
+bool multicore_fifo_wready(void);
 void multicore_fifo_push(uint32_t data);
 uint32_t multicore_fifo_pop_blocking();
 void multicore_fifo_drain();
```

Comment:
Should this be updated to `void multicore_fifo_drain(void);` like you've done with the other functions? :shrug:

Reply:
Yup I'll get that